### PR TITLE
Migrations plugin : document the addition of the baking diff feature

### PR DESCRIPTION
**Pull request by HavokInspiration**

### CODE REVIEW

File: fr/migrations.rst [hunk 1]
```diff
@@ -446,6 +446,51 @@ ajoutées au snapshot de votre plugin.
...
+
+Vous pouvez générer un fichier de migrations qui regroupera toutes les
+différences entre deux états de base de données en utilisant le template bake
+``migration_diff``. Pour cela, vous pouvez utiliser la commande suivante::
+
+    $ bin/cake bake migration_diff NameOfTheMigrations
+
+Pour avoir un point de comparaison avec l'état actuel de votre base de données,
+le shell migrations va générer, après chaque appel de ``migrate`` ou
+``rollback`` un fichier "dump". Ce fichier dump est fichier qui contient
```
Comment (cake17) [inline fr/migrations.rst]: est un fichier

File: fr/migrations.rst [hunk 2]
```diff
@@ -446,6 +446,51 @@ ajoutées au snapshot de votre plugin.
...
+l'ensemble de l'état de votre base de données à un point précis dans le temps.
+
+Quand un fichier dump a été généré, toutes les modifications que vous ferez
+directement dans votre SGBD sera ajouté au fichier de migration qui sera généré
```
Comment (cake17) [inline fr/migrations.rst]: seront ajoutées ?

File: fr/migrations.rst [hunk 3]
```diff
@@ -446,6 +446,51 @@ ajoutées au snapshot de votre plugin.
...
+possède déjà un historique de migrations, vous allez avoir besoin de créer le 
+fichier dump manuellement pour qu'il puisse être utilisé comme point de
+comparaison::
+
+    $ bin/cake migrations dump
+
+L'état de votre base de données devra être le même que si vous aviez migré tous
+vos fichiers de migrations avant de créer le fichier dump.
+Une fois que le fichier dump est créé, vous pouvez opérer des changements dans
+votre base de données et utilisez la commande ``bake migration_diff`` quand
```
Comment (cake17) [inline fr/migrations.rst]: et utiliser ?

File: fr/migrations.rst [hunk 4]
```diff
@@ -622,6 +667,26 @@ sous-commande ``seed``::
...
+
+La commande Dump crée un fichier qui sera utilisé avec le template bake
+``migration_diff``::
+
+    $ bin/cake migrations dump
+
+Chaque fichier dump généré est spécifique à la _Connection_ par laquelle il a
+été générée (le nom du fichier est suffixé par ce nom). Cela permet à la 
+commande ``bake migration_diff`` de calculer le diff correctement dans le cas
+où votre application gérerait plusieurs base de données (qui pourraient être
```
Comment (cake17) [inline fr/migrations.rst]: bases

**Comment by cake17** [comment]
@HavokInspiration great job. I just wrote a few comments on small typos

**Comment by HavokInspiration** [comment]
@cake17 Fixed. I had a weird error on the previous build. Did not know how to fix it. If I get it again I might need your RST knowledge to fix it.

**Comment by HavokInspiration** [comment]
I got the same build error :

```
/data/en/migrations.rst:443: WARNING: Inline literal start-string without end-string.
```

I have no idea what this means or how to fix it.

**Comment by HavokInspiration** [comment]
@antograssiot Thanks for the fix.

**Comment by cake17** [comment]
@antograssiot nice catch

**Comment by ravage84** [comment]
Added some minor fixes. Thanks for your work, @HavokInspiration

**Comment by HavokInspiration** [comment]
Thanks @ravage84.
It does indeed look better. 
Anymore comments ? Once merged, I'll release migrations 1.6.0

**Pull request merged**